src/main/java/org/kainos/ea/api/JobCapabilityService.java

```diff
@@ -44,4 +47,19 @@ public List<JobCapability> getAllCapabilities() throws FailedToGetJobCapabilityE
             throw new FailedToGetJobCapabilityException();
         }
     }
+
+    public int addCapability(CapabilityRequest capabilityRequest) throws JobCapabilityNotAddedException,
+            FailedToAddJobCapabilityException {
+        try {
```

Comment:
Is there any validation you need to do? Limits in the SQL table?

Reply:
validation now added, no names over 70 chars allowed